src/types/index.ts

```
@@ -57,8 +57,8 @@ export type Country = {
   timezones: string[];
   continents: string[];
   flags: { png: string; svg: string; alt?: string };
-  coatOfArms: { png?: string; svg?: string };
+  coatOfArms: { png: string; svg: string; alt?: string } | {};
   startOfWeek: string;
-  capitalInfo: { latlng?: [number, number] };
-  postalCode?: { format: string; regex?: string };
+  capitalInfo: { latlng: [number, number] | [] } | {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)